app/pb_kits/playbook/pb_body/_body.html.erb

```diff
@@ -2,5 +2,5 @@
   id: object.id,
   data: object.data,
   class: object.classname) do %>
-  <%= !object.children.nil? ? capture(&object.children) : object.text %>
+  <%= !object.children.nil? ? capture(&object.children) : object.content %>
```

Comment:
It's usually easier on the reader if we don't use negated logic, how about:

```rb
<%= object.children.nil? ? object.content : capture(&object.children) %>
```

or

```rb
<%= object.children.present? ? capture(&object.children) : object.content %>
```